test/docfx.Test/lib/JsonSchemaTest.cs

```diff
@@ -223,12 +216,25 @@ public void TestJsonSchemaConfirmance(string description, string schemaText, str
         [InlineData("{'contains': {'const': 1}}", "[2]",
             @"['warning','array-contains-failed','Array '' should contain at least one item that matches JSON schema','file',1,1]")]
 
+        // additionalItems validation
+        [InlineData("{'items': [{'const': 1}], 'additionalItems': true}", "[1]", "")]
+        [InlineData("{'items': [{'const': 1}], 'additionalItems': false}", "[1]", "")]
+        [InlineData("{'items': [{'const': 1}], 'additionalItems': false}", "[1, 2]",
+            "['warning','array-length-invalid','Array '' length should be <= 1','file',1,1]")]
+        [InlineData("{'items': [{'const': 1}], 'additionalItems': {'const': 2}}", "[1, 3]",
+            "['warning','invalid-value','Invalid value for '': '3'','file',1,5]")]
+
         // required validation
         [InlineData("{'required': []}", "{}", "")]
         [InlineData("{'required': ['a']}", "{'a': 1}", "")]
         [InlineData("{'required': ['a']}", "{'b': 1}",
             "['warning','missing-attribute','Missing required attribute: 'a'','file',1,1]")]
 
+        // boolean schema
+        [InlineData("true", "[]", "")]
```

Comment:
why we add same content in different PRs?

Reply:
yes, this PR is based on #4904, (has a dependency), when #4094 is merged, the diff will disappear.